documentation/source/troubleshooting.rst

```diff
@@ -144,3 +144,16 @@ When you install cocotb, ``pip`` uses the system (or some other) compiler that s
 But when you try to run cocotb with the older Questa, it prepends the older libraries Questa ships with to ``LD_LIBRARY_PATH``.
 This causes the older libstdc++ Questa ships with to be loaded, resuling in the error message.
 For Questa, you can use the ``-noautoldlibpath`` option to turn off the ``LD_LIBRARY_PATH`` prepend to resolve this issue.
+
+
+Using cocotb with more than one Python installation
+===================================================
+
+Users of cocotb with more than one Python installation must take care not to re-use cached versions of the installed cocotb package.
```

Comment:
I would specifically call out conda here, some people aren't not aware of how conda works.

```suggestion
Users of cocotb with more than one Python installation (including conda env users) must take care not to re-use cached versions of the installed cocotb package.
```